services/distribution/src/main/resources/application.yaml

```diff
@@ -105,7 +110,7 @@ services:
         download-timeout-in-seconds: ${KEY_DOWNLOAD_ANDROID_DOWNLOAD_TIMEOUT_IN_SECONDS:30}
         overall-timeout-in-seconds: ${KEY_DOWNLOAD_ANDROID_OVERALL_TIMEOUT_IN_SECONDS:480}
       ios-exposure-detection-parameters:
```

Comment:
where can this be set vor v2 only?

Reply:
There is only one set of Key Download paramters. It was not specified that there should be separate. Is that the case?